robot.py

```diff
@@ -601,10 +595,10 @@ def teleopDrivetrain(self):
             self.drivetrain.setWheelLock(False)
 
         if (driver.getYButton()):
-            # drive so that the center of the bot is on the center of the note, without rotating
-            self.drivetrain.alignWithNote(0, 0, None)
-            return False
-
+            self.drivetrain.alignWithNote(None, None, 0)
+            self.drivetrain.driveStraight(math.max(self.deadzoneCorrection(driver.getLeftY() * translational_clutch, 0), self.driver.deadzone))
+            # self.drivetrain.alignWithNote(0, 0, None)
+            return
```

Comment:
Does this still need `return False`?

Reply:
I think it needs a return of some sort to prevent calling drivetrain.execute twice

Reply:
It used to have `return False`, that's why I asked